src/log-pcap.c

```diff
@@ -1307,6 +1343,10 @@ static OutputInitResult PcapLogInitCtx(SCConfNode *conf)
     int en;
     PCRE2_SIZE eo = 0;
 
+    if (g_pcap_data) {
+        FatalError("A pcap-log instance is already active, only one can be enabled.");
+    }
```

Comment:
Happy to drop this comment, its not really related to this task. I created a ticket with some of the issues here: https://redmine.openinfosecfoundation.org/issues/7640